### PR TITLE
Update redis and kafka doc

### PR DESCRIPTION
**Pull request by moarychan**
Update docs.

Fixes 
- https://github.com/Azure/azure-sdk-for-java/issues/25937
- https://github.com/Azure/azure-sdk-for-java/issues/25938

### CODE REVIEW

File: docs/src/main/asciidoc/authentication.adoc [hunk 1]
```diff
@@ -144,8 +145,13 @@ Following are the Azure built-in roles for authorizing access to Azure services
 |link:{azure-storage-queue-data-reader}[Storage Queue Data Reader]
 |Read and list Azure Storage queues and queue messages.
 
+|link:{azure-redis-cache-contributor}[Redis Cache Contributor]
+|Manage Redis caches.
+
 |===
 
+When using Spring Cloud Azure Resource Manager to get the connection strings of Event Hubs, Service Bus, and Storage Queue, and properties of Cache for Redis, assign the Azure built-in role `Contributor`. Azure Cache for Redis is special, and you can also assign the `Redis Cache Contributor` role to get the Redis properties.
```
Comment (saragluna) [inline docs/src/main/asciidoc/authentication.adoc:153]: Should we follow the same format as line #24, to add NOTE: before this?

File: docs/src/main/asciidoc/kafka-support.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [#kafka-support]
 == Kafka Support
 
-Connect to Azure Event Hubs using Spring Kafka libraries. There are two approaches to connect to Azure Event Hubs for Kafka, the first one is to provide the Azure Event Hubs connection string directly, the other is to use Azure Resource Manager to retrieve the connection string.
+Connect to Azure Event Hubs(Basic pricing tier is not supported) using Spring Kafka libraries. There are two approaches to connect to Azure Event Hubs for Kafka, the first one is to provide the Azure Event Hubs connection string directly, the other is to use Azure Resource Manager to retrieve the connection string.
```
Comment (saragluna) [inline docs/src/main/asciidoc/kafka-support.adoc:4]: add a space before `(`

Comment (saragluna) [reply docs/src/main/asciidoc/kafka-support.adoc:4]: Maybe also add a link to the pricing tier description / support for Event Hubs here.

**Review by saragluna — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/authentication.adoc [hunk 3]
```diff
@@ -150,7 +150,7 @@ Following are the Azure built-in roles for authorizing access to Azure services
 
 |===
 
-When using Spring Cloud Azure Resource Manager to get the connection strings of Event Hubs, Service Bus, and Storage Queue, and properties of Cache for Redis, assign the Azure built-in role `Contributor`. Azure Cache for Redis is special, and you can also assign the `Redis Cache Contributor` role to get the Redis properties.
+NOTE: When using Spring Cloud Azure Resource Manager to get the connection strings of Event Hubs, Service Bus, and Storage Queue, and properties of Cache for Redis, assign the Azure built-in role `Contributor`. Azure Cache for Redis is special, and you can also assign the `Redis Cache Contributor` role to get the Redis properties.
```
Comment (saragluna) [inline docs/src/main/asciidoc/authentication.adoc:153]: get the connection strings of Event Hubs, Service Bus, and Storage Queue, **or** properties of Cache for Redis

**Review by saragluna — APPROVED** [review]
LGTM.

File: docs/src/main/asciidoc/authentication.adoc (hunk 3 above)
Comment (moarychan) [reply docs/src/main/asciidoc/authentication.adoc]: Thanks.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**